test/end-to-end-testapp/settings.js

```diff
@@ -4,7 +4,8 @@ var SDKsettings = {
     initialize, this may be apiKey, projectId, primaryCustomerType, etc. These are passed
     into the src/initialization.js file as the
     */
-    appId: 'testAppId',
+    appId: '1518360323',
```

Comment:
Is this a real appId? If it is, I don't think you should be committing it and just leave it as `testAppId`